traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1063,6 +1063,7 @@ sub sleep_timer {
 sub process_config_files {
 
 	( $log_level >> $INFO ) && print "\nINFO: ======== Start processing config files ========\n";
+	$ats_uid          = getpwnam("ats");
```

Comment:
Where does $ats_uid get used next?  Looks like you pulled it into a local variable but I don't see where you refer to it.

Reply:
@dewrich It was defined as a global before, which I retained. It is used in function `replace_cfg_file` and in `backup_file` which is called by `process_config_files`.  Should I just locally define $ats_uid in those 2 functions? Looks like it is also redefined locally in `adv_processing_udev`.